Directory.Build.targets

```diff
@@ -34,4 +45,29 @@
     <PackageReference Include="Microsoft.DotNet.XUnitExtensions" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(IsSourceProject)' == 'true'">
```

Comment:
https://github.com/dotnet/runtime/blob/43813ac73242fa78c463d456bf755e3a6622b5d7/eng/versioning.targets#L26-L28 is missing here

Reply:
It's already showing up in all the DLLs without me adding this.

Reply:
See my comment: https://github.com/dotnet/maintenance-packages/pull/135#issuecomment-2418190786

Reply:
Yes, but when you set `GenerateNeutralResourcesLanguageAttribute` which I'm recommending in another comment, you will nee to add this.

Reply:
Oh, this one: https://github.com/dotnet/maintenance-packages/pull/135#issuecomment-2420106423

Reply:
I had to refresh the page to see it.